python/tools/create_inbox_message/create_inbox_message.py

```diff
@@ -0,0 +1,33 @@
+# :coding: utf-8
+# :copyright: Copyright (c) 2024 ftrack
+
+import ftrack_api
+
```

Comment:
@seebergs please import and use logging to notify user about action success.

Reply:
Added.